doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_pm_optimization.rst

```diff
@@ -0,0 +1,197 @@
+|Type           | List of the        | Source of data to  | Time of restoration| Software module          |
+|               | peripherals        | restore            |                    | responsible for restoring|
++===============+====================+====================+====================+==========================+
+|Active         | * ``MVDMA``        | Device driver's    | Decided by the     | The device driver        |
+|peripherals    |                    | code and data      | driver             |                          |
++---------------+--------------------+--------------------+--------------------+--------------------------+
+
+Peripherals specific to the Application Domain
+----------------------------------------------
+
+There are no peripherals specific to the Application Domain.
+
+Peripherals specific to the Secure Domain
+-----------------------------------------
+
+The Secure Domain contains additional peripherals that require handling in the *Local Suspend to RAM* state.
+
++---------------+--------------------+--------------------+--------------------+--------------------------+
+|Type           | List of the        | Source of data to  | Time of restoration| Software module          |
+|               | peripherals        | restore            |                    | responsible for restoring|
++===============+====================+====================+====================+==========================+
+|Active         | * ``CRACEN``       | Device driver's    | Decided by the     | The device driver        |
+|peripherals    |                    | code and data      | driver             |                          |
++---------------+--------------------+--------------------+--------------------+--------------------------+
+
+Peripherals specific to the Radio Domain
+----------------------------------------
+
+The Radio Domain does not implement the *Local Suspend to RAM* state.
+
+Optimization example
+********************
+
+This example focuses on a HID device use case (for example, a mouse)
+
+A mouse has typically two main application states:
+
+* Active, when the user is moving the mouse.
+* Idle, triggered after an inactivity timeout until an input from the user is detected.
+
+The following is an example workflow for the *Active* application state:
+
+  1. Sample input data from sensors.
+  2. Construct a HID report.
+  3. Transmit the report over the radio.
+  4. Sleep until the next period (usually, less than 1 ms).
+
+Power consumption targets
+=========================
+
+The power consumption targets for this example are the following:
+
+* Sleep currents (5 V supply):
+
+  * Application core (Suspend-to-RAM): < 10 µA
+  * Radio core (Suspend-to-Idle, cache disabled): < 10 µA
+
+* Active currents (estimates at 5 V):
+
+  * Radio core in idle: ~0.25 mA
+  * App core in idle: ~0.15 mA
+  * Core wake-up every 1 ms: +0.05 mA
+  * ADC sample every 1 ms: +0.10 mA
+  * SPI transaction every 1 ms: +0.20 mA
```

Comment:
Consider adding note here that the SPI can be reduced to 0.03 mA by using nrfy API. Alternatively, don't call it "power consumption targets", but "power consumption measurements".